.github/workflows/merge-queue.yml

```
@@ -13,193 +18,147 @@
+    if: needs.prepare.outputs.TEST_CHUNKS
+    runs-on: arc-test
+    timeout-minutes: 45
+    env:
+      AFFECTED_PROJECTS: ${{ matrix.projects }}
+      NX_BASE: ${{ needs.prepare.outputs.NX_BASE }}
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.prepare.outputs.TEST_CHUNKS) }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/load-deps
```

actionlint (1.7.4)

missing input "yarn_cache_folder" which is required by action "Load Dependencies" defined at "./.github/actions/load-deps". all required inputs are "cache_key", "cypress_cache_folder", "generated_files", "yarn_cache_folder"

(action)